tests/sentry/api_gateway/test_proxy.py

```diff
@@ -0,0 +1,40 @@
+from unittest.mock import patch
+
+import responses
+from django.test.client import RequestFactory
+from pytest import raises
+from rest_framework.exceptions import NotFound
+
+from sentry.api_gateway.proxy import proxy_request
+from sentry.testutils.helpers.api_gateway import SENTRY_REGION_CONFIG, ApiGatewayTestCase
+from sentry.utils import json
+
+
+class ProxyTestCase(ApiGatewayTestCase):
+    @responses.activate
+    @patch("sentry.types.region.get_region_for_organization")
+    def test_simple(self, region_fnc_patch):
+        request = RequestFactory().get("http://sentry.io/get")
```

Comment:
```suggestion
        request = RequestFactory().get("http://sentry.io/get?a=1&a=2&b=3")
```

You'll probably want to test if the query strings are properly propagated.